jobs/python/sdk/README.md

```diff
@@ -0,0 +1,210 @@
+
+> **Note:** This example leverages the Python SDK. If you are looking for the example using only HTTP requests, [click here](../http/).
+
+This quickstart includes two apps:
+
+- `job-scheduler/app.py`, responsible for scheduling, retrieving and deleting jobs.
+- `job-service/app.py`, responsible for handling the triggered jobs.
+
+## Prerequisites
+
+- [Python 3.8+](https://www.python.org/downloads/)
+- [Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+- [Initialized Dapr environment](https://docs.dapr.io/getting-started/install-dapr-selfhost/)
+
+## Run all apps with multi-app run template file
+
+This section shows how to run both applications at once using [multi-app run template files](https://docs.dapr.io/developing-applications/local-development/multi-app-dapr-run/multi-app-overview/) with `dapr run -f .`. This enables you to test the interactions between multiple applications and will `schedule`, `run`, `get`, and `delete` jobs within a single process.
+
+1. Build the apps:
+
+<!-- STEP
+name: Install python dependencies
+-->
+
+```bash
+pip3 install -r requirements.txt
+```
+
+<!-- END_STEP -->
+
+2. Run the multi app run template:
+
+<!-- STEP
+name: Run multi app run template
+expected_stdout_lines:
+  - '== APP - job-scheduler == Sending request to schedule job: R2-D2'
+  - '== APP - job-service == Scheduling job: R2-D2'
+  - '== APP - job-service == Job scheduled: R2-D2'
+  - '== APP - job-scheduler == Response: {"name":"R2-D2","job":"Oil Change","dueTime":15}'
+  - '== APP - job-scheduler == Sending request to retrieve job: R2-D2'
+  - '== APP - job-service == Retrieving job: R2-D2'
+  - '== APP - job-scheduler == Job details for R2-D2: {"name":"R2-D2","due_time":"15s","data":{"droid":"R2-D2","task":"Oil Change"}}'
+  - '== APP - job-scheduler == Sending request to schedule job: C-3PO'
+  - '== APP - job-service == Scheduling job: C-3PO'
+  - '== APP - job-service == Job scheduled: C-3PO'
+  - '== APP - job-service == Starting droid: R2-D2'
+  - '== APP - job-service == Executing maintenance job: Oil Change'
+  - '== APP - job-scheduler == Response: {"name":"C-3PO","job":"Limb Calibration","dueTime":20}'
+  - '== APP - job-scheduler == Sending request to retrieve job: C-3PO'
+  - '== APP - job-service == Retrieving job: C-3PO'
+  - '== APP - job-scheduler == Job details for C-3PO: {"name":"C-3PO","due_time":"20s","data":{"droid":"C-3PO","task":"Limb Calibration"}}'
+expected_stderr_lines:
+output_match_mode: substring
+match_order: none
+background: true
+sleep: 60
+timeout_seconds: 120
+-->
+
+```bash
+dapr run -f .
+```
+
+The terminal console output should look similar to this, where:
+
+- The `R2-D2` job is being scheduled.
+- The `R2-D2` job is being retrieved.
+- The `C-3PO` job is being scheduled.
+- The `C-3PO` job is being retrieved.
+- The `R2-D2` job is being executed after 15 seconds.
+- The `C-3PO` job is being executed after 20 seconds.
+
+```text
```

Comment:
All these need to be changed to job-scheduler-sdk and job-service-sdk

Reply:
ok